client/extensions/woocommerce/app/orders/index.js

```diff
@@ -15,10 +15,10 @@ import OrdersList from './orders-list';
 
 class Orders extends Component {
 	render() {
-		const { className, site } = this.props;
+		const { className, site, translate } = this.props;
 		return (
 			<Main className={ className }>
-				<OrderHeader siteSlug={ site ? site.slug : false } />
+				<OrderHeader siteSlug={ site ? site.slug : false } breadcrumbs={ ( <span>{ translate( 'Orders' ) }</span> ) } />
```

Comment:
:disappointed: 99 existing translations will be lost with this change.
If the string must change, the following option has already been translated **87** times:
`translate( 'Order:' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).